### ISSUE TITLE
Loss function for re-training UnCRtainTS on the mono-temporal dataset.

### ISSUE THREAD

**Issue by kongdebug** [PatrickTUM/UnCRtainTS#4]
Hello @PatrickTUM ,
Thank you for your great work!

I want to know what the loss function used when re-training UnCRtainTS on the mono-temporal SEN12MS-CR dataset.

**Comment by PatrickTUM** [comment]
Hi @kongdebug ,

thanks, glad you like it! The loss function for UnCRtainTS on SEN12MS-CR is a simple L2 loss.

The reasoning is that we first evaluated all considered architectures on the two datasets using mean prediction only, in order to identify which architecture is best and to establish the benefits of the proposed architecture. Afterwards, we focused on the multi-temporal dataset and using the NLL loss with the best architecture. L2 was chosen due to the Gaussian generalizing it, but in case you only care about mean prediction then you may also be interested in considering other losses, such as L1 or SSIM --- which we also found beneficial but didn't investigate any further in the context of this publication. Hoping this helps you!

Cheers,
Patrick
